### DISCUSSION TITLE
Continuing execution of a script

### DISCUSSION THREAD

**Discussion by jordi-z80** [sebastienros/jint#1849] [Q&A] [answered]
Hi!

I'm looking to replace my custom made scripting language with Jint, and my main concern is to know if engine "executions" can be continued.

In example, I run an script with a 50_000 iterations for with a log(i) , and use MaxStatements (4000) options. Running the code, prints about 1000 numbers and stops, which is great.

Now on the next frame (it's a game) I'd like to run 4000 instructions more, so that it prints from 1000 to 2000.

Is this possible? Any examples available?

Thanks!

**Comment by lahma** [comment, marked as answer]
Max statements is used to prevent script for running too long / use too much resources. Maybe you could try if the debugger functionality and breakpoints could help you somehow.
